custom_components/ocpp/api.py

```diff
@@ -793,23 +796,27 @@ def on_meter_values(self, connector_id: int, meter_value: Dict, **kwargs):
             processed_keys = []
             for idx, sv in enumerate(bucket[om.sampled_value.name]):
                 if m in sv and om.phase.value not in sv:
-                    self._metrics[sv[m]] = round(float(sv[om.value.value]), 1)
+                    self._metrics[sv[m]].value = round(float(sv[om.value.value]), 1)
                     if om.unit.value in sv:
                         if sv[om.unit.value] == DEFAULT_POWER_UNIT:
```

Comment:
Not as a result of your changes but in reviewing I've noticed there should be a line here to set the units:
`self._metrics[sv[m]].unit = sv[om.unit.value]`

Reply:
ok, I'll add that